packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -432,6 +432,8 @@ private void applySettings(Map<String, Object> settings) {
           break;
         case "gestureNavigationEnabled":
           break;
+        case "bouncesEnabled":
+          break;
```

Comment:
This should be implemented for Android as well (https://developer.android.com/reference/android/webkit/WebView#setOverScrollMode(int))

Reply:
The implementation should be trivial, but per https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#platform-support since we know what the interface should look like it would be fine to land this as an iOS-only change if you aren't comfortable implementing it for Android.